README.md

```diff
@@ -11,6 +11,7 @@ We want to create a set of services/systems to permit multiple tenants to flexib
 [design]: docs/esi-design.md
 [reqs]: docs/esi-requirements.md
 [upstream]: docs/upstream-features.md
+[deployment]: docs/deployment.md
```

Comment:
You want to actually modify the bullet list (above these links)  to include a link to the deployment doc.

Reply:
Whoops - done.